packages/orchestrator/internal/template/server/main.go

```diff
@@ -75,8 +77,17 @@ func New(
 	if err != nil {
 		return nil, fmt.Errorf("failed to create build metrics: %w", err)
 	}
+
+	featureFlags, err := featureflags.NewClient()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create feature flags client: %w", err)
+	}
+
+	sandboxFactory := sandbox.NewFactory(networkPool, devicePool, featureFlags, defaultAllowSandboxInternet)
```

Comment:
do you need it here as well? Why not to use the one from `packages/orchestrator/main.go`